### PR TITLE
WideStance Left and right 

### PR DESCRIPTION
**Pull request by larskna**
## Why? What?

We want to jump to the right and left whith the WideStance, so we can better catch the blall 
Its only for the keeper
Fixes #

## ToDo / Known Issues

The Motienfile is in this moment indentical with the WideStance. That is only the .json file 

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test

You have to kick the ball to the Keeper (playernumber 1) and look

### CODE REVIEW

File: crates/control/src/behavior/defend.rs [hunk 1]
```diff
@@ -58,24 +58,38 @@ impl<'cycle> Defend<'cycle> {
...
+            (true, _, _) => Some(MotionCommand::WideStance {
+                direction: JumpDirection::Center,
+            }),
+            (false, true, _) => Some(MotionCommand::WideStance {
+                direction: JumpDirection::Left,
+            }),
+            (false, _, true) => Some(MotionCommand::WideStance {
+                direction: JumpDirection::Right,
+            }),
+            (false, false, false) => None,
```
Comment (schluis) [inline crates/control/src/behavior/defend.rs:92]: I think a simple if-else statement is completely sufficient. This is basically a glorified if-else. Other opinions? @h3ndrk

Comment (h3ndrk) [reply crates/control/src/behavior/defend.rs:92]: Looks like it, but not sure about the `_` in line 89 :thinking:

Comment (schluis) [reply crates/control/src/behavior/defend.rs:92]: I'm sure. All cases are mutually exclusive

**Comment by oleflb** [comment]
@larskna whats the state here?

**Comment by larskna** [comment]
It can be close. 
it was the first iteration of the keeper jump. I will do a new PR for that

**Pull request closed without merging**